### PR TITLE
Update device config API to include controller information like enterprise name/id

### PR DESCRIPTION
**Pull request by zedi-pramodh**
Fetch DeviceName, DeviceId, ProjectId, ProjectName, EnterpriseName and EnterpriseId

The purpose of these is to be able to share with applications at http://169.254.169.254/

Signed-off-by: Pramodh Pallapothu <pramodh@zededa.com>

### CODE REVIEW

File: api/proto/config/devconfig.proto [hunk 1]
```diff
@@ -136,6 +136,13 @@ message EdgeDevConfig {
   // power on capability; power off can be done locally using the Local Profile
   // Server API.
   DeviceOpsCmd shutdown = 34;
+
+  string device_name = 35;
+  string device_id = 36;
```
Comment (eriknordmark) [inline api/proto/config/devconfig.proto:141]: That is already in the message; I think it is called "id" at the top of the EdgeDevConfig.

**Review by eriknordmark — CHANGES_REQUESTED** [review]
Please remove the duplicate device_id field.
Also, please split into two commits; one with the proto file change and a second with the results of make proto proto-vendor; the latter should have at least one file modified in pkg/pillar/vendor/

**Comment by zedi-pramodh** [comment]
Sure will remove device_id and also submit another commit for proto-vendor

**Review by eriknordmark — APPROVED** [review]
LGTM

**Pull request merged**